medusa/clients/torrent/generic.py

```diff
@@ -232,66 +202,47 @@ def send_torrent(self, result):
         :return:
         :rtype: str or bool
         """
-        r_code = False
 
-        log.debug('Calling {name} Client', {'name': self.name})
 
-        if not self.auth:
-            if not self._get_auth():
```

Comment:
i'm pretty sure that there are clients that need this check/call for authentication. Have you verified this with all the torrent clients?

Reply:
Yes all current clients have that function available, that's why I removed it ( I don't like superfluous code), but feel free to add it in again.